src/app/shared/shared.module.ts

```diff
@@ -17,7 +65,55 @@ export function httpFactory(backend: XHRBackend, options: RequestOptions, inject
 
 @NgModule({
     imports:[MsgBoxModule],
-    exports:[FormsModule, MsgBoxModule]
+    exports:[
+        CommonModule,
+        FormsModule, 
+        ReactiveFormsModule,
+        MsgBoxModule,
+        ButtonModule, 
+        DataTableModule, 
+        PanelModule, 
+        OverlayPanelModule,  
+        ChartModule, 
+        CardModule, 
+        TabViewModule, 
+        InputTextModule, 
+        InputSwitchModule, 
+        DropMenuModule, 
+        DialogModule, 
+        FormModule,
+        MultiSelectModule, 
+        GrowlModule ,
+        DropdownModule,
+        InputTextareaModule,
+        TooltipModule,
+        TableModule,
+        FieldsetModule,
+        DataViewModule,
+        ScrollPanelModule,
+        DataScrollerModule,
+        TreeModule,
+        ProgressBarModule,
+        ContextMenuModule,
+        ConfirmDialogModule,
+        CheckboxModule, 
+        SplitButtonModule, 
+        SpinnerModule,
+        MessageModule,
+        ClipboardModule,
+        RadioButtonModule,
+        SelectButtonModule,
+        BreadcrumbModule,
+        HomeDialogModule,
+        SidebarModule,
+        CalendarModule,
+        InplaceModule,
+        ChipsModule,
+        DeferModule,
+        AccordionModule,
+        BadgeModule,
+        PasswordModule
```

Comment:
Password is a specific function.. not  used across multiple modules .  Is it right to place it in shared module.

Reply:
This PasswordModule is a UI module and not a business logic one. 
This is the [PasswordModule](https://www.primefaces.org/primeng/v5.2.7/#/password) from PrimeNG.
If you want to provide password fields in the UI we can use this instead of a input field of type password.